apps/SageAccountWeb/docker/Caddyfile

```diff
@@ -0,0 +1,32 @@
+# The Caddyfile is an easy way to configure your Caddy web server.
+#
+# Unless the file starts with a global options block, the first
+# uncommented line is always the address of your site.
+#
+# To use your own domain name (with automatic HTTPS), first make
+# sure your domain's A/AAAA DNS records are properly pointed to
+# this machine's public IP, then replace ":80" below with your
+# domain name.
+
+:80 {
+	# Set this path to your site's directory.
+	root * /usr/share/caddy
+
+	# Enable the static file server.
+	file_server
+
+	# Another common task is to set up a reverse proxy:
+	# reverse_proxy localhost:8080
+
+	# Or serve a PHP site through php-fpm:
+	# php_fastcgi localhost:9000
+
+	# Modify default Caddyfile to handle SPA routing -- redirect all requests to index.html
+	handle {
+		try_files {path} {path}/ /index.html
+		file_server
+	}
```

Comment:
This is the only change to the default...for some reason I couldn't manage to do this with the base nginx image so I decided to use [Caddy](https://github.com/caddyserver/caddy) instead

Reply:
hello caddy